src/tabs/tabs.scss

```diff
@@ -112,6 +111,32 @@
   }
 }
 
+@mixin iui-tabs-horizontal {
+  flex-direction: column;
+
+  .iui-tabs {
+    display: flex;
+  }
+}
+
+@mixin iui-tabs-vertical {
+  flex-direction: row;
+
+  .iui-tabs {
+    display: inline-flex;
+    flex-direction: column;
+
+    li,
+    .iui-tab {
+      width: 100%;
+    }
+
+    ~ .iui-tabs-content {
+      flex-grow: 1;
+    }
+  }
+}
+
 @mixin iui-tab-not-animated {
   .iui-tab {
     @media (prefers-reduced-motion: no-preference) {
```

Comment:
It seems like that one rule is overriding another:
![image](https://user-images.githubusercontent.com/36186912/129674657-168017d2-d225-4262-813c-0aab15d824cf.png)

Reply:
I think this is fixed now, the box-shadow transition was leftover code.